packages/experimental/Button/src/Button.types.ts

```diff
@@ -1,34 +1,15 @@
 import * as React from 'react';
 import { ViewProps, ViewStyle, ColorValue } from 'react-native';
 import { TextProps } from '@fluentui-react-native/experimental-text';
-import { FontTokens, IBorderTokens, IShadowTokens } from '@fluentui-react-native/tokens';
+import { FontTokens, IBorderTokens, IColorTokens, IShadowTokens, LayoutTokens } from '@fluentui-react-native/tokens';
 import { IFocusable, IPressableHooks, IWithPressableOptions } from '@fluentui-react-native/interactive-hooks';
 import type { IViewWin32Props } from '@office-iss/react-native-win32';
 import { IconProps, IconSourcesType } from '@fluentui-react-native/icon';
 
 export const buttonName = 'Button';
+export type ButtonSize = 'small' | 'medium' | 'large';
 
-export interface ButtonTokens extends FontTokens, IBorderTokens, IShadowTokens {
-  /**
-   * Background color for the button
-   */
-  backgroundColor?: ColorValue;
```

Comment:
The tokens I deleted can be handled by the other tokens that I added to the "extends" portion (IColorTokens, LayoutTokens). It just seemed better to make ButtonTokens inherit from them in case new ones are added in the future, it wouldn't be something the ButtonTokens has to add separately.